pvdeg/utilities.py

```diff
@@ -416,3 +416,72 @@ def ts_gid_df(file, gid):
                 res.lat = meta.latitude[gid]
                 res.lon = meta.longitude[gid]
     return res
+
+
+def tilt_azimuth_scan(
+    weather_df=None,
+    meta=None,
+    weather_kwarg=None,
+    tilt_count=18,
+    azimuth_count=72,
+    sky_model="isotropic",
+    temp_model="sapm",
+    conf_0="insulated_back_glass_polymer",
+    conf_inf="open_rack_glass_polymer",
+    T98=70, # [°C]
+    x_0=6.5, # [cm]
+    wind_speed_factor=1.7):
+
+    """
+    Calculate a minimum standoff distance for roof mounded PV systems as a function of tilt and azimuth.
+
+    Parameters
+    ----------
+    weather_df : pd.DataFrame
+        Weather data for a single location.
+    meta : pd.DataFrame
+        Meta data for a single location.
+    tilt_count : integer
+        Step in degrees of change in tilt angle of PV system between calculations.
+    azimuth_count : integer
+        Step in degrees of change in Azimuth angle of PV system relative to north.
+    sky_model : str, optional
+        Options: 'isotropic', 'klucher', 'haydavies', 'reindl', 'king', 'perez'.
+    temp_model : str, optional
+        Options: 'sapm'.  'pvsyst' and 'faiman' will be added later.
+        Performs the calculations for the cell temperature.
+    conf_0 : str, optional
+        Default: 'insulated_back_glass_polymer'
+    conf_inf : str, optional
+        Default: 'open_rack_glass_polymer'
+    x_0 : float, optional
+        Thermal decay constant [cm], [Kempe, PVSC Proceedings 2023]
+    wind_speed_factor : float, optional
+        Wind speed correction factor to account for different wind speed measurement heights
+        between weather database (e.g. NSRDB) and the tempeature model (e.g. SAPM)
+        The NSRD provides calculations at 2m (i.e module height) but SAPM uses a 10m height.
+        It is recommended that a power-law relationship between height and wind speed of 0.33 be used.
+        This results in a wind_speed_factor of 1.7. It is acknowledged that this can vary significantly. 
+    Returns
+        standoff_series : 2-D array with each row consiting of tilt, azimuth, then standoff
+    """
+    
+    standoff_series=np.array([(azimuth_count+1)*(tilt_count+1)][3])
+    for x in range (0, azimuth_count+1):
+        for y in range (0,tilt_count+1):
+            standoff_series[x+y][0]=azimuth_count*180/azimuth_count
+            standoff_series[x+y][1]=tilt_count*90/azimuth_count
+            standoff_series[x+y][2]=standards.standoff(
```

Comment:
standards not imported

Reply:
This is a place holded that is just not finished yet. it shouldn't be brought in. I'm hoping to get to this shortly this week.

Reply:
okay, I'll remove it from the current PR and we include it once it's ready